packages/components/src/utils/headless-cms/service.tsx

```diff
@@ -40,6 +40,8 @@ export const getAllArticles = async (
         locale: node.language.code.toLowerCase(),
         slug: node.slug,
       });
+      // TODO: fix any
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Some RCHC builds are just bad and this needs to be changed quite often. Hopefully we find something that sticks.

Reply:
Ah, this TODO was no longer needed, it was before the `i18n.ts` -type definition fix.